app/controllers/video_papers_controller.rb

```diff
@@ -13,7 +13,7 @@ def index
       order_by = VideoPaper.order_by(params[:order_by])
       if params[:user]
         user = User.find(params[:user].to_i)
-        @video_papers = user.my_video_papers.paginate :page=>params[:page], :per_page=>10, :order=>order_by
+        @video_papers = user.my_video_papers.paginate(:page=>params[:page], :per_page=>10).order(order_by)

```

Comment:
Did you test this out? I think it will work. But it seems possible the `order` needs to come before the paginate. Otherwise it might just be ordering the pages instead of the whole query. To be save perhaps just switch the order to be before the paginate.

Reply:
From https://github.com/mislav/will_paginate:

```
New in Active Record 3:

# paginate in Active Record now returns a Relation
Post.where(:published => true).paginate(:page => params[:page]).order('id DESC')
```

we are using `activerecord (4.1.16)` according to the Gemfile.lock